pkg/ui/workspaces/cluster-ui/src/jobs/util/duration.spec.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+import React from "react";
+import { cockroach } from "@cockroachlabs/crdb-protobuf-client";
+import { render, screen } from "@testing-library/react";
+import { makeTimestamp } from "src/util";
+import {
+  JOB_STATUS_RUNNING,
+  ALL_JOB_STATUSES,
+  JOB_STATUS_SUCCEEDED,
+} from "./jobOptions";
+import { Duration } from "./duration";
+
+// Job running for 10 minutes
+const START_SECONDS = 0;
+const CURRENT_SECONDS = 600;
+
+describe("<Duration>", () => {
+  describe("when job is not running or completed", () => {
```

Comment:
nit: We try to keep tests as flat as possible (no nested describes or even describes at all) because it makes it easier to manage scope when there is only one scope. These tests are pretty simple, so there isn't a lot to worry about, but for more complicated test suites, it makes a big difference.

https://cockroachlabs.slack.com/archives/C0182T8NUH0/p1678124832350459 goes into more details, but the TLDR is that the file describes what tests are there.